templates/book_borrowing_dashboard.html

```diff
@@ -0,0 +1,101 @@
+{% extends "base.html" %}
+{% block title %}Contact{% endblock %}
+{% block content %}
+  <div class="cover-container d-flex h-80 p-3 mx-auto flex-column">
+      <main role="main">
+            <div class="book_borrowing_dashboard">
+                <div >
+                    <button id="reserv_button" type="button" class="btn btn-primary book_dashboard_button">
+                        Reserved : {{ num_of_reserved }}</button>
+                    <button id="booked_button" type="button" class="btn btn-primary book_dashboard_button">
+                        Borrowed : {{ num_of_borrowed }}</button>
+                </div>
+                <table id="reserved_table" class="table booking_dashboard">
+                  <thead>
+                    <tr>
+                        <th scope="col">Title</th>
+                        <th scope="col">Author</th>
+                        <th scope="col">Reservation date</th>
+                        <th scope="col">Reservation expiry date</th>
+                    </tr>
+                  </thead>
+                  <tbody>
+                  {% if reserved_books == [] %}
+                    <tr>
+                        <td>------</td>
+                        <td>------</td>
+                        <td>------</td>
+                        <td>------</td>
+                    </tr>
+                  {% else %}
+                  {% for item in reserved_books %}
+                    <tr>
+                        <td>
+                            {{item[0].title}}
+                            {% if item[0].__tablename__ == "magazines" %}
```

Comment:
You could also just do here something like that: `{% if item.type == 'magazine' %}` - more readable.

Reply:
Yes, item[0].__tablename__ looks ...weird :)